splice_si_api/src/main/java/com/splicemachine/si/impl/store/CompletedTxnCacheSupplier.java

```diff
@@ -35,8 +35,8 @@
  *         Date: 6/18/14
  */
 public class CompletedTxnCacheSupplier implements TxnSupplier{
+    private volatile TxnView[] cache;
```

Comment:
Is it possible to not make it volatile? Maybe by implementing `invalidate` like `Arrays.fill(cache, null);`

Reply:
Changed in commit https://github.com/splicemachine/spliceengine/pull/5103/commits/74c7644dcfb2a08f831039cd2ed377d61fc3adb9